core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/run_sh.go

```diff
@@ -276,7 +276,7 @@ func (builtin *RunShCapabilities) Execute(ctx context.Context, _ *builtin_argume
 	if err != nil {
 		return "", stacktrace.Propagate(err, "error occurred while preparing the sh command to execute on the image")
 	}
-	fullCommandToRun := []string{shellWrapperCommand, "-c", commandToRun}
+	fullCommandToRun := []string{shellWrapperCommand, "-c", fmt.Sprintf("%v \"$(%v)\" %v %v", "printf \"%s\\n\"", commandToRun, ">>", "/proc/1/fd/1")}
```

Comment:
worst case if this err's can do a retry with `[]string{shellWrapperCommand, "-c", commandToRun}`?

Reply:
https://github.com/kurtosis-tech/avalanche-package/blob/main/src/node_launcher.star#L47-L122 how about this approach

we wrap up the exec and make it write to some file in a logs dir and tail the logs there; but piping to fd1 makes sense too

Reply:
sounds good will try this

Reply:
implemented this and it works for streaming logs!